waffle-mock-contract/src/index.ts

```diff
@@ -50,5 +52,22 @@ export async function deployMockContract(wallet: Wallet, abi: ABI): Promise<Mock
   const mockedContract = new Contract(mockContractInstance.address, abi, wallet) as MockContract;
   mockedContract.mock = mock;
 
+  mockedContract.staticcall = async (contract: Contract, functionName: string, ...params: any) => {
+    const fn = contract.interface.functions[functionName];
```

Comment:
This also means that you can call it like this: `mockERC20.staticcall(amIRich, 'check')` as opposed to `mockERC20.staticcall(amIRich, 'check()')`, right? We should probably document that fact somehow.

Reply:
Yup; that's the case.  I'll add that to the readme and a test case.